### DISCUSSION TITLE
Add ForceTeleport method (Originally requested as ForceResync)

### DISCUSSION THREAD

**Discussion by hudmarc** [FirstGearGames/FishNet#265] [Feature Request]
There was a ForceResend method implemented, and this is great. However, the intent of my original issue https://github.com/FirstGearGames/FishNet/pull/164 was to have ForceResync momentarily reset the instant rates and force the NetworkTransform to jump to the latest received Transform data. It has nothing to do with forcing a send of transform data,  because if the NT is extremely desynchronized on a client and Teleport is set to off there is no way to force it to teleport to its most recent TransformData and you are instead forced to wait as the NT slowly rubberbands towards its intended position.
https://github.com/FirstGearGames/FishNet/pull/164/commits/00cfe372bf07343cf3724818840d4785e0403fa8
The original change I suggested just adds a boolean called forceResync which has the exact same effect as turning on _enableTeleport except it is temporary and is reset to false as soon as the teleport occurs, and it can be called on clients.

**Comment by hudmarc** [comment]
Actually after looking at the NT code some more, this could be as easy as setting the Teleport variable public.

**Comment by FirstGearGames** [comment]
Teleport is set on the component and there's no 'force teleport' option because you could set the value to something low, like 1 unit, and it would automatically teleport. 1 unit might not sound like a large enough distance but if you consider that most objects only move about a 20th of a unit per tick it's plenty. You can of course, adjust the value.
